Sources/ApolloCodegenLib/JSON.swift

```diff
@@ -59,18 +59,18 @@ public enum JSONValue: Codable, Equatable {
     
     switch self {
     case .dictionary(let dictionary):
-        guard let directValue = dictionary[currentKey] else {
-            throw JSONValueError.noValueForKey(currentKey)
-        }
-        
-        let remainingKeys = Array(keyPath.dropFirst())
-        guard !remainingKeys.isEmpty else {
-            return directValue
-        }
-        
-        return try directValue.valueForKeyPath(remainingKeys)
+      guard let directValue = dictionary[currentKey] else {
+        throw JSONValueError.noValueForKey(currentKey)
+      }
+
+      let remainingKeys = Array(keyPath.dropFirst())
+      guard !remainingKeys.isEmpty else {
```

Comment:
you can probably use `remainingKeys.apollo.isNotEmpty` here if you find that more readable (I do, which is why I made it)

Reply:
Haha cool. I just fixed auto-indent on this file, didn't even look to improve it.